sdk/python/symbolchain/nem/TransactionFactory.py

```diff
@@ -1,32 +1,49 @@
+from symbolchain import nc
+
+from ..BasicTransactionFactory import BasicTransactionFactory
 from ..BufferWriter import BufferWriter
-from ..TransactionDescriptorProcessor import TransactionDescriptorProcessor
-from .ImportanceTransferTransaction import ImportanceTransferTransaction
-from .TransferTransaction import TransferTransaction
+from ..ByteArray import ByteArray
+from ..CryptoTypes import Hash256, PublicKey, Signature
+from .ExtendedTypeParsingRules import extend_type_parsing_rules
+from .Network import Address
+
+
+def nem_type_converter(value):
+	if isinstance(value, Address):
+		# yes, unfortunatelly, nem's Address is 40 bytes string, but we need to pass it as actual bytes not to confuse ByteArray
+		return nc.Address(str(value).encode('utf8'))
```

Comment:
🤢

Reply:
can you elaborate on why string doesn't work ?

Reply:
(nc.)Address is ByteArray, and ByteArray ctor, looks like this:
```py
		raw_bytes = array_input
		if isinstance(raw_bytes, str):
			raw_bytes = unhexlify(raw_bytes)
```